### PR TITLE
Improve resource cleanup section

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

Made a few improvements to the resource cleanup section: 

* Fixed spelling of "failcount" to "fail count"
* Updated `crm resource list` to `crm resource status`
* Added info about fail counts not being cleaned automatically by default
* Improved the crmsh procedure, esp. by adding a command to actually see the fail count

### PR creator: Are there any relevant issues/feature requests?

* bsc#1211019
* jsc#DOCTEAM-977

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP5
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
  - [x] 15 SP1
- SLE-HA 12
  - [ ] 12 SP5
  - [ ] 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by tahliar** [comment]
Fail count info: 

image

Improved procedure:

image

File: xml/ha_resource_constraints.xml [hunk 1]
```diff
@@ -838,12 +838,12 @@
      for resource <literal>rsc1</literal> to preferably run on
      <literal>&node1;</literal>. If it fails there,
      <literal>migration-threshold</literal> is checked and compared to the
-     failcount. If failcount &gt;= migration-threshold then the resource is
+     fail count. If fail count &gt;= migration-threshold then the resource is
```
Comment (dariavladykina) [inline xml/ha_resource_constraints.xml:841]:
```suggestion
     fail count. If <literal>failcount</literal> &gt;= migration-threshold, then the resource is
```

File: xml/ha_resource_constraints.xml [hunk 2]
```diff
@@ -944,19 +944,19 @@
     <title>Specifying resource failover nodes with &crmsh;</title>
     <para>
      To determine a resource failover, use the meta attribute
-     <literal>migration-threshold</literal>. If failcount exceeds
+     <literal>migration-threshold</literal>. If fail count exceeds
```
Comment (dariavladykina) [inline xml/ha_resource_constraints.xml:947]:
```suggestion
     <literal>migration-threshold</literal>. If the fail count exceeds
```

File: xml/ha_resource_constraints.xml (hunk 1 above)
Comment (dariavladykina) [reply xml/ha_resource_constraints.xml:841]: I just made it identical to line 955. Please discard if not relevant.

File: xml/ha_resource_constraints.xml [hunk 3]
```diff
@@ -944,19 +944,19 @@
...
      <literal>migration-threshold</literal> on all nodes, the resource
      remains stopped. For example:
     </para>
  <screen>&prompt.crm.conf;<command>location rsc1-&node1; rsc1 100: &node1;</command></screen>
     <para>
      Normally, <literal>rsc1</literal> prefers to run on <literal>&node1;</literal>.
      If it fails there, <literal>migration-threshold</literal> is checked and compared
-     to the failcount. If <literal>failcount</literal> &gt;= <literal>migration-threshold</literal>
+     to the fail count. If <literal>failcount</literal> &gt;= <literal>migration-threshold</literal>
```
Comment (dariavladykina) [inline xml/ha_resource_constraints.xml:955]:
```suggestion
     to the fail count. If <literal>failcount</literal> &gt;= <literal>migration-threshold</literal>,
```

File: xml/geo_booth_i.xml [hunk 4]
```diff
@@ -734,7 +734,7 @@ ticket = "tkt-sap-prod" <xref linkend="co-ha-geo-booth-config-ticket" xrefstyle=
      cluster site. That means, it checks if the cluster is healthy enough to
      run the resource (all resource dependencies are fulfilled, the cluster
      partition has quorum, no dirty nodes, etc.). For example, if a service in
-     the dependency-chain has a failcount of <literal>INFINITY</literal> on all
+     the dependency-chain has a fail count of <literal>INFINITY</literal> on all
```
Comment (dariavladykina) [inline xml/geo_booth_i.xml:737]:
```suggestion
     the dependency chain has a fail count of <literal>INFINITY</literal> on all
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

**Pull request merged**